### DISCUSSION TITLE
Collections on Mobile

### DISCUSSION THREAD

**Discussion by GodGamer9000** [ppy/osu#22915] [Q&A]
I was trying to add a song to a collection on iOS and I was not sure how. Am I missing something, or is this functionality that is not there right now for iOS?

Edit: Did not find people talking about collections specifically, but found people talking about right clicking which is basically the same problem.

**Comment by Joehuu** [comment]
You can do this for now (note for hovering you have to swipe vertically on the collection menu item or "here" in the video to prevent it from being clicked):

https://user-images.githubusercontent.com/35318437/226199374-93404984-ab80-487c-9d53-0549a3c1038c.mp4
